mdn/yari
# MP-9 Integrate Glean.js MDN

## Summary

This PR adds the [Glean.js](https://github.com/mozilla/glean.js/) library to enable first party analytics on MDN.  (https://mozilla-hub.atlassian.net/browse/MP-9) 



### Problem

We want to move away from using third party analytics and have better visibility into MDN internally.



### Solution

This PR adds the Glean.js library to MDN and will be used to verify it for further use. 

### Testing.

To test locally do the following

-  Be sure to build first build the glean ping definitions using 
`      yarn build:glean`
- To enable glean and 'ping' logging add to your .env : 
```
REACT_APP_GLEAN_CHANNEL=stage
REACT_APP_GLEAN_DEBUG=true
REACT_APP_GLEAN_ENABLED=true
```
- Navigate to any docs page on your localhost and you should see pings being logged in the dev tools.
- To verify data is being receieved on the server check out 
   -  https://debug-ping-preview.firebaseapp.com/pings/mdn-dev
